forge-gui/res/adventure/Shandalar/custom_cards/nahiris_armory.txt

```diff
@@ -0,0 +1,7 @@
+Name:Nahiri's Armory
+ManaCost:no cost
+Types:Artifact
+S:Mode$ ReduceCost | ValidCard$ Card | ValidSpell$ Activated.Equip | Activator$ You | Amount$ 1 | EffectZone$ Command | Description$ Equip costs you pay cost {1} less.
+T:Mode$ AttackersDeclared | ValidAttackers$ Creature.modified+YouCtrl | TriggerZones$ Command | Execute$ TrigConjure | TriggerDescription$ Whenever a modified creature you control attacks, conjure a card from Nahiri's Armory into your hand
```

Comment:
I'm unaware of any cards that do not currently specify these effects as coming from a spellbook. So as awkward as it is, I think we should probably reword this to "Nahiri's Armory's spellbook".